test/e2e/pod_ps_test.go

```diff
@@ -212,17 +212,17 @@ var _ = Describe("Podman ps", func() {
 		Expect(ec).To(Equal(0))
 
 		_, ec, podid2 := podmanTest.CreatePodWithLabels("", map[string]string{
-			"io.podman.test.label": "value1",
-			"io.podman.test.key":   "irrelevant-value",
+			"app":                "myapp",
+			"io.podman.test.key": "irrelevant-value",
 		})
 		Expect(ec).To(Equal(0))
 
 		_, ec, podid3 := podmanTest.CreatePodWithLabels("", map[string]string{
-			"io.podman.test.label": "value2",
+			"app": "test",
 		})
 		Expect(ec).To(Equal(0))
 
-		session := podmanTest.Podman([]string{"pod", "ps", "--no-trunc", "--filter", "label=io.podman.test.key", "--filter", "label=io.podman.test.label=value1"})
+		session := podmanTest.Podman([]string{"pod", "ps", "--no-trunc", "--filter", "label=app", "--filter", "label=app=myapp"})
```

Comment:
I understand why this test was passing now! I didn't notice that the two filters in the old test were for different label keys. `label=...key` and `label=...label=value1`. This test was passing because there is only one pod that has a `key` label and a `label` label, not because the value matched.

With that in mind, my change actually changes the meaning of this test. I'm not sure what the intention of the old test was, but it didn't test filtering on label values. The new test does that, but maybe it misses something else. Thoughts?

Reply:
In case anyone is curious, I found this by breaking out my trusty debugger.

1. I ran the debugger with `dlv exec bin/podman -- pod ps --no-trunc --filter label=io.podman.test.key --filter label=io.podman.test.label=value1`, copying the command line arguments from the test exactly.
2. I set a breakpoint on the following line (just after the filter command line arg processing): https://github.com/containers/podman/blob/a6b300ef7ee19da0c590af9bbcd1a35de7fa8c84/cmd/podman/pods/ps.go#L83
3. I ran `print psInput.Filters` in the debugger. I expected to see something like `["io.podman.test.label", "io.podman.test.label"]`, but that's not what I saw! Instead, this was the output:
  ```
map[string][]string [
	"label": [
		"io.podman.test.key",
		"io.podman.test.label",
	], 
]
```

I was surprised to see filters for two different labels, which made me realize that I had made a mistake when reading the test code earlier.